apps/web/app/layout.tsx

```diff
@@ -153,7 +162,12 @@ export default async function RootLayout({ children }: { children: React.ReactNo
             "/insights",
           ]}
         />
-        <Providers>{children}</Providers>
+
+        <Providers>
+          <I18nProvider translations={translations} locale={locale} ns={ns}>
```

Comment:
At first, I tried drilling `t` function itself, but it threw 
```
Error: Functions cannot be passed directly to Client Components ...
```

So, I am instead passing `translations`